strawberry/subscriptions/protocols/graphql_ws/handlers.py

```diff
@@ -190,7 +190,8 @@ async def handle_async_results(
         await self.send_message(GQL_COMPLETE, operation_id, None)
 
     async def cleanup_operation(self, operation_id: str) -> None:
-        await self.subscriptions[operation_id].aclose()
+        with suppress(RuntimeError):
+            await self.subscriptions[operation_id].aclose()
```

Comment:
when does this error happen?

Reply:
See [this SO](https://stackoverflow.com/questions/72204244/python-asynchronous-generator-is-already-running) 
I think it is because we use it here as well
https://github.com/strawberry-graphql/strawberry/pull/3570/files/b67bcb6df86cc864ba63c8ebd424c0ff11039289#diff-9a6420babd3ea891d8956d6313ec8f3e15e3682dd65051d5a4a1d80c0be27826R167

Reply:
It only failed in 3.3 though. We also do the same in graphql-ws-transport
https://github.com/strawberry-graphql/strawberry/blob/fce504dfe7a896670064a8b4d2a8dc80a697501d/strawberry/subscriptions/protocols/graphql_transport_ws/handlers.py#L297

Not that I'm saying its ideal...

Reply:
is it failing consistently now?

Reply:
just on 3.3 yeah. on every run.

Reply:
@DoctorJohn maybe you can double check this? 😊

Reply:
This was added in PR #2794 without a test because the author said it's due to a race condition. Will check